NEWS

```diff
@@ -48,6 +48,12 @@ CHANGES WITH 253 in spe:
         * systemd-fstab-generator now supports x-systemd.makefs option for
           /sysroot (in the initrd).
 
+        * New Network param SocketPriority to assign the Linux socket priority
```

Comment:
suffix `SocketPriority=` with `=` please, as we usually do

Reply:
Fixed in the squashed commit.